packages/components/psammead-inline-link/src/index.stories.jsx

```diff
@@ -24,4 +55,98 @@ storiesOf('Components|InlineLink', module)
       </>
     ),
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'inline-link inside a regular paragraph',
+    ({ text, longText, script, service }) => (
+      <>
+        <RegularParagraph script={script} service={service}>
+          {longText}{' '}
+          <InlineLink href="https://www.bbc.com/news">{text}</InlineLink>{' '}
+          {longText}
+        </RegularParagraph>
+      </>
+    ),
+    { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'inline-link inside a bold paragraph',
+    ({ text, longText, script, service }) => (
+      <>
+        <BoldParagraph script={script} service={service}>
+          {longText}{' '}
+          <InlineLink href="https://www.bbc.com/news">{text}</InlineLink>{' '}
+          {longText}
+        </BoldParagraph>
+      </>
+    ),
+    { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'inline-link inside a italic paragraph',
+    ({ text, longText, script, service }) => (
+      <>
+        <ItalicParagraph script={script} service={service}>
+          {longText}{' '}
+          <InlineLink href="https://www.bbc.com/news">{text}</InlineLink>{' '}
+          {longText}
+        </ItalicParagraph>
+      </>
+    ),
+    { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'inline-link inside a bold and italic paragraph',
+    ({ text, longText, script, service }) => (
+      <>
+        <BoldItalicParagraph script={script} service={service}>
+          {longText}{' '}
+          <InlineLink href="https://www.bbc.com/news">{text}</InlineLink>{' '}
+          {longText}
+        </BoldItalicParagraph>
+      </>
+    ),
+    { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'experimental styled inline link with text-decoration style',
+    ({ text, longText, script, service }) => (
+      <>
+        <RegularParagraph script={script} service={service}>
+          {longText}{' '}
+          <InlineLink
+            style={{
+              borderBottom: 'none',
+              textDecoration: `underline ${C_POSTBOX}`,
+            }}
+            href="https://www.bbc.com/news"
+          >
+            {text}
+          </InlineLink>{' '}
+          {longText}
+        </RegularParagraph>
+        <Caption script={script} service={service}>
```

Comment:
We should extract this into a separate function as well just to make it more readable.